frontend/src/metabase/nav/containers/MainNavbar/BookmarkList/sortable.css

```diff
@@ -0,0 +1,9 @@
+.react-sortable-hoc-helper.sorting {
```

Comment:
Could we move this to emotion? E.g. 

const Component = styled.div```
  .sorting {
    
  }
```;

Reply:
@ranquild It may be possible, but I think it would be a lot of work for little gain, and maybe not possible.

This class is added by `react-sortable-hoc` to the cloned dragged component, when we are dragging.

And it's not something, afaik, we can write a custom component and inject styled-components/emotion.

Also this library has been deprecated for a while now, so even less incentive to invest in this I suppose.

Reply:
I don't think you would need to change anything in how `sortable` works. Just use this `sorting` class in your `emotion` styles:

```
export const SidebarBookmarkItem = styled(...)`
  &.sorting {
    background-color: ${color("white")};
    border-bottom: 2px solid ${color("brand")};
  }
`;
```